app/javascript/groceries/components/CheckInItem.vue

```diff
@@ -15,10 +15,11 @@ li.flex.items-center.break-word.mb-2(
       span(v-if='item.needed > 1') {{' '}} ({{item.needed}})
     span {{ ' ' }}
     ElTooltip(
+      v-if="isSpouseItem(item)"
```

Comment:
(Unrelated change. Noticed an issue when running the groceries feature specs. See commit: https://github.com/davidrunger/david_runger/pull/6047/commits/a0fa300045e9963d3ed8b4e5a9f4d426b2c77356 .)